contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -97,17 +97,20 @@ abstract contract GovernorTimelockAccess is Governor {
      * delayed since queuing, and an array indicating which of the proposal actions will be executed indirectly through
      * the associated {AccessManager}.
```

Comment:
```suggestion
     * delayed since queuing, an array indicating which of the proposal actions will be executed indirectly through
     * the associated {AccessManager}, and another indicating which will be scheduled during {queue}. Note that
     * those that must be scheduled are cancellable by `AccessManager` guardians.
```